src/main/java/com/minecolonies/entity/pathfinding/Pathfinding.java

```diff
@@ -217,17 +217,17 @@ private static void renderDebugText(@NotNull Node n, float f1)
         GL11.glDepthMask(false);
 
         Tessellator tessellator = Tessellator.getInstance();
-        WorldRenderer worldrenderer = tessellator.getWorldRenderer();
+        VertexBuffer VertexBuffer = tessellator.getBuffer();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00117)